synapse/lib/stormtypes.py

```diff
@@ -1646,8 +1646,26 @@ def getObjLocals(self):
             'genr': self._libGenr,
             'list': self._libList,
             'ingest': self._libIngest,
+            'fromAxon': self._fromAxon,
         }
 
+    async def _fromAxon(self, sha256):
+        '''
+        Use the feed API to load a syn.nodes formatted export from the axon.
+
+        Args:
+            sha256 (str): The sha256 of the file saved in the axon.
+
+        Returns:
+            int: The number of nodes loaded.
+        '''
+        sha256 = await tostr(sha256)
+        opts = {
+            'user': self.runt.snap.user.iden,
+            'view': self.runt.snap.view.iden,
+        }
+        return await self.runt.snap.core.feedFromAxon(sha256, opts=opts)
```

Comment:
should this be a dyncall?

Reply:
No, because the API is exposed via telepath.  Calling it on a remote or local cortex is the same.